### PR TITLE
Updating xblock cookiecutter

### PR DESCRIPTION
**Pull request by jinder1s**
**Description:**

Updating cookiecutter-xblock by using python-template. This does not fix the docker for the xblock. And there will be another PR with tests for cookiecutter-xblock.

**JIRA:**

[XXX-XXXX](https://openedx.atlassian.net/browse/XXX-XXXX)

**Merge checklist:**
- [ ] All reviewers approved
- [x] CI build is green
- [x] Changelog record added
- [ ] Documentation updated (not only docstrings)
- [ ] Commits are squashed

**Post merge:**
- [ ] Delete working branch (if not needed anymore)

### CODE REVIEW

File: python-template/{{cookiecutter.placeholder_repo_name}}/setup.py [hunk 1]
```diff
@@ -82,6 +100,13 @@ def is_requirement(line):
     keywords='Python edx',
     classifiers=[
         'Development Status :: 3 - Alpha',
+        {%- if cookiecutter.supports_django == "yes" %}
```
Comment (feanil) [inline python-template/{{cookiecutter.placeholder_repo_name}}/setup.py:103]: Is it more accurate to say that in requires django than to say it supports django?

Comment (jinder1s) [reply python-template/{{cookiecutter.placeholder_repo_name}}/setup.py:103]: yep, you are correct

File: cookiecutter-xblock/hooks/post_gen_project.py [hunk 2]
```diff
@@ -0,0 +1,87 @@
+"""
+Post-generation cookiecutter hook.
+
+* See docs/decisions/0003-layered-cookiecutter.rst
+"""
+import os
+import shutil
```
Comment (dianakhuang) [inline cookiecutter-xblock/hooks/post_gen_project.py:7]: I'm not sure how much you want to try to make changes here, but a lot of this seems like boilerplate that could be consolidated. But maybe that's just an issue with the way we're hacking cookiecutter, and this is actually the cleanest solution.

Comment (jinder1s) [reply cookiecutter-xblock/hooks/post_gen_project.py:7]: I've almost decided to pull the plug on consolidation of boiler plate into one. Previously, I didn't do that cause I wanted people to be able to create a new repo by calling: "cookiecutter git_url --directory cookeicutter". But now that we've decided to put all tests in parent directory and require edx-lint to be installed for a successfull build, that is no longer really as straight forward as I'd want. So yes, consolidation will probably happen soonish.

**Pull request merged**